contrib/packs/actions/setup_virtualenv.yaml

```diff
@@ -29,3 +29,9 @@
       required: false
       description: Action timeout in seconds. Action will get killed if it doesn't finish in timeout
       type: integer
+    index_url:
+      default: "https://pypi.org/simple"
```

Comment:
This default means all users that do not use this option will have `-i https://pypi.org/simple` in the pip command which seems pointless.